io_scene_niftools/utils/bones_mapper.py

```diff
@@ -0,0 +1,21 @@
+
+    # Dictionnary to map bones names to id
+    bonename_dict = defaultdict(int)
+    id = 0
+    for x in bone_names:
+        if not bonename_dict[x]:
```

Comment:
Are you expecting some bone names to be duplicated in the `bone_names` list? If not, it is probably not necessary to have this check, and you could use `enumerate` for the id.

Reply:
No indeed, the enumerate do the job and it's now the case, thanks